webdriver/tests/get_element_shadow_root/conftest.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+@pytest.fixture
+def get_checkbox_dom(inline):
```

Comment:
Maybe remove the `get_` prefix here because it's not returning a method.

Reply:
Done in eda55bd